### ISSUE TITLE
Regression in 0.17: We can't have a page called `index.md` in the same folder as an index section in …

### ISSUE THREAD

**Issue by jplatte** [Keats/gutenberg#2140]
The site https://github.com/matrix-org/matrix.org/tree/zola builds just fine on 0.16.1, but fails to build on 0.17.0 and 0.17.1. It fails with

```
Error: We can't have a page called `index.md` in the same folder as an index section in "…/matrix.org/content"
```

There are no overlapping index files as far as I can tell.

Probably related to #2101, but that was fixed in 0.17.1 which doesn't fix this bug.

**Comment by Keats** [comment]
There is another bug fix in the `next` branch. I'll try the site with it as soon as I can

**Comment by Keats** [comment]
```
(venv) ~/C/p/matrix.org (zola|✔) [0|SIGINT]$ zola build
Building site...
Checking all internal links with anchors.
> Successfully checked 35 internal link(s) with anchors.
-> Creating 678 pages (1 orphan) and 120 sections
Done in 7.0s.
```

with the `next` branch so it should be good! I'll release 0.17.2 this week, just waiting a few more days to see if there are other issues.

**Comment by jplatte** [comment]
Fixed in 0.17.2, thanks!
